src/main/java/ap/ims/config/IMSSecurityConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package ap.ims.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+public class IMSSecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    @Override
+    public void configure(final HttpSecurity http) throws Exception {
+        http.csrf().disable()
+            .authorizeRequests()
+            // Allow access to basic actuator endpoints
+            .antMatchers("/manage/info*", "/manage/health*").permitAll()
+            // Allow access to all api endpoints
+            .antMatchers("/api/**").permitAll()
+            .anyRequest().authenticated()
+            .and()
+            .httpBasic();
+    }
+}
```

Comment:
Basic auth is on for most things.  Anything under "/api" does not require authentication, and the /info and /health endpoints do not require it either.  Everything else does require authentication.  The root project application.yml sets the credentials as user: admin | pass: admin.  

We can adjust this as necessary.  Maybe the "/api" should be locked down as well.  We discussed this very briefly with Jeff and Landon Thursday.